app/services/save_measurements.rb

```diff
@@ -135,8 +142,13 @@ def save(streams)
         .logger.warn "Stream.import failed for: #{import.failed_instances.inspect}"
     end
 
+    created_sessions = Session.where(id: session_ids)
```

Comment:
I wanted to somehow optimize this logic, cause it takes a significant amount of time if there is a lot of sessions to create, but there is not a lot after the first pull so optimization is not a must-be here.